workspaces/azure-storage-explorer/plugins/azure-storage/README.md

```diff
@@ -0,0 +1,155 @@
+# Azure Storage
+
+Welcome to the `AzureStorage` plugin!
+
+## Intruduction
+
+This plugin helps to explore (navigate and download) the blobs in Azure storage account.
+
+![1](docs/1.png)
```

Comment:
Can we give these some nice descriptions? That would help those with screen readers 👍